### PR TITLE
Update 04-visualization-ggplot2.Rmd

### PR DESCRIPTION
**Pull request by erinmgraham**
To address Plotting with ggplot2 --> geom_boxplot + geom_jitter note #803
Replace geom_boxplot(alpha=0) with geom_boxplot(outlier.shape = NA)

### CODE REVIEW

**Comment by sstevens2** [comment]
Nice PR!  I'd recommend maybe adding an inline comment or some additional text to to the lesson explaining why we added this argument.

**Comment by erinmgraham** [comment]
Thanks for the tip Sarah! I'm still fairly new at this and made my edit on the patch-1 bratch. Is that correct?

File: 04-visualization-ggplot2.Rmd [hunk 1]
```diff
@@ -271,10 +271,15 @@ measurements and of their distribution:
...
 ggplot(data = surveys_complete, mapping = aes(x = species_id, y = weight)) +
-    geom_boxplot(alpha = 0) +
+    geom_boxplot(outlier.shape = NA) +
     geom_jitter(alpha = 0.3, color = "tomato")
 ```
 
+Note that combining the geom_boxplot and geom_jitter layers together can cause 
+outlier points to be plotted twice - once as outliers for the boxplot and once 
+again by jitter. To prevent duplication of points in this example, the geom_boxplot 
+layer outlier.shape argument is set to NA.
```
Comment (sstevens2) [inline 04-visualization-ggplot2.Rmd:281]:
```suggestion
layer `outlier.shape` argument is set to NA.
```

**Comment by sstevens2** [comment]
Yes @erinmgraham, nice job editing your previous branch used in this PR!  I left a small formatting suggestion for the argument text.  Looks great!

**Comment by erinmgraham** [comment]
Suggestion accepted! Thanks for your help 😊

**Review by sstevens2 — APPROVED** [review]
Looks great!  Hopefully we've left it in a good place that the @datacarpentry/r-ecology-lesson-maintainers are happy with.

**Comment by ErinBecker** [comment]
Thank you for this contribution @erinmgraham. This is a great fix to #803. Unfortunately we have had a gap in Maintainer activity on this lesson. I am filling in to take care of some PRs in preparation for [transition to the new lesson infrastructure](https://carpentries.github.io/workbench/faq.html#update-fork-from-styles). I neglected to notice that your PR was an earlier submitted fix for #803 than one that I already accepted and merged https://github.com/datacarpentry/R-ecology-lesson/pull/817. We are currently bringing new Maintainers on for this lesson to improve the contributor experience and ensure useful PRs like yours are merged in a timely manner. Please accept my apologies.

**Pull request closed without merging**